lib/sparse/src/common/sparse_vector.rs

```diff
@@ -9,7 +9,8 @@ use crate::common::types::{DimId, DimWeight};
 /// - indices to be unique
 /// - indices and values to be the same length
 /// - indices and values to be non-empty
-/// - indices to be sorted TODO(sparse) this is not validated yet
+/// - values can be any f32
```

Comment:
This comment line may be unnecessary. Even more, `NaN` is a valid `f32` value but we don't expect nan (and +- inf too)

Reply:
Sure I can remove it.